src/slices/filters.ts

```diff
@@ -47,6 +57,34 @@ const reducers = {
       });
     });
   },
+  toggleFilterOption: (state: FiltersState, { payload }: PayloadAction<FilterPayload>) => {
+    if (!state.static) {
+      state.static = {};
+    }
+    const { filterCollectionId, filter: targetFilter, shouldSelect } = payload;
+    if (!filterCollectionId) {
+      console.warn(`invalid static filters id: ${filterCollectionId}`);
+      return;
+    }
+    const filterCollection = state.static[filterCollectionId];
+    const foundFilter = filterCollection?.find(storedSelectableFilter => {
+      const storedFilter = storedSelectableFilter.filter;
+      return storedFilter.fieldId === targetFilter.fieldId
```

Comment:
in the future maybe we can export a filter equality method or something as part of AnswersUtilities (not right now just thinking it might be useful in the future for people)

Reply:
sounds good